src/handlers/notification/index.js

```diff
@@ -74,21 +74,17 @@ const processMessage = async (msg) => {
     throw new Error('Invalid message received from kafka')
   }
   const { metadata, from, to, content } = msg.value
-  const { action, status } = metadata.event
-  if (action === 'prepare' && status === 'success') {
+  const {status} = metadata.event
+  if (status === 'success') {
     return Callback.sendCallback(Config.DFSP_URLS[to].transfers, 'post', content.headers, content.payload).catch(err => {
       Logger.error(`error posting to the callback - ${err}`)
       throw err
     })
-  } else if (action === 'prepare' && status !== 'success') {
+  } else {
```

Comment:
In the else part too, we need to have logic specific based on action. In future when we handle negative scenarios, more specific rules need to be applied. So better to have if/else logic to include action.